www/docs/server/router.md

```diff
@@ -2,7 +2,7 @@
 id: router
 title: Define Routers
 sidebar_label: Define Routers
-slug: /router
+slug: /server/router
```

Comment:
routers here or procedure above?

might as well be consistent with singular/plural

Reply:
Great shout, yes!